src/modules/aws_ecr/rpcs/build.ts

```diff
@@ -140,18 +56,27 @@ export class EcrBuildRpc extends RpcBase {
     githubRef: string,
     githubPersonalAccessToken: string,
   ): Promise<RpcResponseObject<typeof this.outputTable>[]> => {
-    await this.ensureAwsIamModule(_dbId);
+    await this.ensureModules(_dbId);
```

Comment:
can we use `dbId` instead of `_dbId`? Let's follow the convention that when the parameter is prefixed with `_` is because it is not in use.

Reply:
Sure.